libfdt: Rename the package to pylibfdt

kirelagin opened the pull request:
The “canonical” name of the Python libfdt package, i.e. the one used on PyPI and, hence, the one that is listed in the requirements of the dependent modules is `pylibfdt`, but the upstream build system produced a package called `libfdt`.

Given that, realistically, everyone will be using `pylibfdt`, as seen on PyPI, our choices are to either patch / jailbreak the requirement in all dependent packages, or follow PyPI and rename the package itself.

Rename the package once and for all, since it is easier than modifying every single dependent one.

## Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [Nixpkgs 25.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/doc/release-notes/rl-2511.section.md) (or backporting [24.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/release-notes/rl-2411.section.md) and [25.05](https://github.com/NixOS/nixpkgs/blob/master/doc/manual/release-notes/rl-2505.section.md) Nixpkgs Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
- [NixOS 25.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2511.section.md) (or backporting [24.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) and [25.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) NixOS Release notes)
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md), [pkgs/README.md](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md), [maintainers/README.md](https://github.com/NixOS/nixpkgs/blob/master/maintainers/README.md) and other contributing documentation in corresponding paths.

Ping maintainers: @dezgeg for `dtc`/`libfdt`; @sorki and @otavio for the two dependents.

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

kirelagin committed:
libfdt: Rename the package to pylibfdt

The “canonical” name of the Python libfdt package, i.e. the one used on
PyPI and, hence, the one that is listed in the requirements of the
dependent modules is `pylibfdt`, but the upstream build system produced
a package called `libfdt`.

Given that, realistically, everyone will be using `pylibfdt`, as seen on
PyPI, our choices are to either patch / jailbreak the requirement in all
dependent packages, or follow PyPI and rename the package itself.

Rename the package once and for all, since it is easier than modifying
every single dependent one.

Aleksanaa commented:
## `nixpkgs-review` result

Generated using [`nixpkgs-review-gha`](https://github.com/Defelo/nixpkgs-review-gha)

Command: `nixpkgs-review pr 420985`

Logs: https://github.com/Aleksanaa/nixpkgs-review-gha/actions/runs/16116448367


---
### `x86_64-linux` (sandbox = true)
<details>
  <summary>:x: 1 package failed to build:</summary>
  <ul>
    <li>python312Packages.libfdt</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 259 packages built:</summary>
  <ul>
    <li>OVMFFull</li>
    <li>OVMFFull.fd</li>
    <li>alpine-make-vm-image</li>
    <li>argononed</li>
    <li>barbicanclient (python313Packages.python-barbicanclient)</li>
    <li>barbicanclient.dist (python313Packages.python-barbicanclient.dist)</li>
    <li>binwalk</li>
    <li>cloud-init</li>
    <li>cloud-init.dist</li>
    <li>cloud-utils</li>
    <li>cloud-utils.guest</li>
    <li>colima</li>
    <li>cot (python313Packages.cot)</li>
    <li>cot.dist (python313Packages.cot.dist)</li>
    <li>culvert</li>
    <li>debos</li>
    <li>diffoscope</li>
    <li>diffoscope.dist</li>
    <li>diffoscope.man</li>
    <li>diffoscopeMinimal</li>
    <li>diffoscopeMinimal.dist</li>
    <li>diffoscopeMinimal.man</li>
    <li>dt-schema</li>
    <li>dt-schema.dist</li>
    <li>dtc</li>
    <li>fwts</li>
    <li>ghidra-extensions.ghidraninja-ghidra-scripts</li>
    <li>glanceclient (python313Packages.python-glanceclient)</li>
    <li>glanceclient.dist (python313Packages.python-glanceclient.dist)</li>
    <li>gnome-boxes</li>
    <li>goldboot</li>
    <li>heatclient (python313Packages.python-heatclient)</li>
    <li>heatclient.dist (python313Packages.python-heatclient.dist)</li>
    <li>hot-resize</li>
    <li>ironicclient (python313Packages.python-ironicclient)</li>
    <li>ironicclient.dist (python313Packages.python-ironicclient.dist)</li>
    <li>kata-runtime</li>
    <li>libguestfs (python313Packages.guestfs)</li>
    <li>libguestfs.guestfsd (python313Packages.guestfs.guestfsd)</li>
    <li>lima</li>
    <li>lxd-lts</li>
    <li>magnumclient (python313Packages.python-magnumclient)</li>
    <li>magnumclient.dist (python313Packages.python-magnumclient.dist)</li>
    <li>manilaclient (python313Packages.python-manilaclient)</li>
    <li>manilaclient.dist (python313Packages.python-manilaclient.dist)</li>
    <li>mistralclient (python313Packages.python-mistralclient)</li>
    <li>mistralclient.dist (python313Packages.python-mistralclient.dist)</li>
    <li>multipass</li>
    <li>nemu</li>
    <li>open-watcom-bin</li>
    <li>open-watcom-bin-unwrapped</li>
    <li>openstackclient (python313Packages.python-openstackclient)</li>
    <li>openstackclient-full</li>
    <li>openstackclient-full.dist</li>
    <li>openstackclient.dist (python313Packages.python-openstackclient.dist)</li>
    <li>out-of-tree</li>
    <li>python312Packages.cot</li>
    <li>python312Packages.cot.dist</li>
    <li>python312Packages.guestfs</li>
    <li>python312Packages.guestfs.guestfsd</li>
    <li>python312Packages.keystoneauth1</li>
    <li>python312Packages.keystoneauth1.dist</li>
    <li>python312Packages.openstacksdk</li>
    <li>python312Packages.openstacksdk.dist</li>
    <li>python312Packages.openstacksdk.man</li>
    <li>python312Packages.os-client-config</li>
    <li>python312Packages.os-client-config.dist</li>
    <li>python312Packages.osc-lib</li>
    <li>python312Packages.osc-lib.dist</li>
    <li>python312Packages.osc-placement</li>
    <li>python312Packages.osc-placement.dist</li>
    <li>python312Packages.oslo-concurrency</li>
    <li>python312Packages.oslo-concurrency.dist</li>
    <li>python312Packages.oslo-db</li>
    <li>python312Packages.oslo-db.dist</li>
    <li>python312Packages.oslo-log</li>
    <li>python312Packages.oslo-log.dist</li>
    <li>python312Packages.oslo-metrics</li>
    <li>python312Packages.oslo-metrics.dist</li>
    <li>python312Packages.oslo-serialization</li>
    <li>python312Packages.oslo-serialization.dist</li>
    <li>python312Packages.oslo-utils</li>
    <li>python312Packages.oslo-utils.dist</li>
    <li>python312Packages.osprofiler</li>
    <li>python312Packages.osprofiler.dist</li>
    <li>python312Packages.python-aodhclient</li>
    <li>python312Packages.python-aodhclient.dist</li>
    <li>python312Packages.python-barbicanclient</li>
    <li>python312Packages.python-barbicanclient.dist</li>
    <li>python312Packages.python-cinderclient</li>
    <li>python312Packages.python-cinderclient.dist</li>
    <li>python312Packages.python-designateclient</li>
    <li>python312Packages.python-designateclient.dist</li>
    <li>python312Packages.python-glanceclient</li>
    <li>python312Packages.python-glanceclient.dist</li>
    <li>python312Packages.python-heatclient</li>
    <li>python312Packages.python-heatclient.dist</li>
    <li>python312Packages.python-ironicclient</li>
    <li>python312Packages.python-ironicclient.dist</li>
    <li>python312Packages.python-keystoneclient</li>
    <li>python312Packages.python-keystoneclient.dist</li>
    <li>python312Packages.python-magnumclient</li>
    <li>python312Packages.python-magnumclient.dist</li>
    <li>python312Packages.python-manilaclient</li>
    <li>python312Packages.python-manilaclient.dist</li>
    <li>python312Packages.python-mistralclient</li>
    <li>python312Packages.python-mistralclient.dist</li>
    <li>python312Packages.python-neutronclient</li>
    <li>python312Packages.python-neutronclient.dist</li>
    <li>python312Packages.python-novaclient</li>
    <li>python312Packages.python-novaclient.dist</li>
    <li>python312Packages.python-octaviaclient</li>
    <li>python312Packages.python-octaviaclient.dist</li>
    <li>python312Packages.python-openstackclient</li>
    <li>python312Packages.python-openstackclient.dist</li>
    <li>python312Packages.python-swiftclient</li>
    <li>python312Packages.python-swiftclient.dist</li>
    <li>python312Packages.python-troveclient</li>
    <li>python312Packages.python-troveclient.dist</li>
    <li>python312Packages.python-watcherclient</li>
    <li>python312Packages.python-watcherclient.dist</li>
    <li>python312Packages.python-zaqarclient</li>
    <li>python312Packages.python-zaqarclient.dist</li>
    <li>python312Packages.python-zunclient</li>
    <li>python312Packages.python-zunclient.dist</li>
    <li>python312Packages.tempest</li>
    <li>python312Packages.tempest.dist</li>
    <li>python313Packages.keystoneauth1</li>
    <li>python313Packages.keystoneauth1.dist</li>
    <li>python313Packages.libfdt</li>
    <li>python313Packages.openstacksdk</li>
    <li>python313Packages.openstacksdk.dist</li>
    <li>python313Packages.openstacksdk.man</li>
    <li>python313Packages.os-client-config</li>
    <li>python313Packages.os-client-config.dist</li>
    <li>python313Packages.osc-lib</li>
    <li>python313Packages.osc-lib.dist</li>
    <li>python313Packages.osc-placement</li>
    <li>python313Packages.osc-placement.dist</li>
    <li>python313Packages.oslo-concurrency</li>
    <li>python313Packages.oslo-concurrency.dist</li>
    <li>python313Packages.oslo-db</li>
    <li>python313Packages.oslo-db.dist</li>
    <li>python313Packages.oslo-log</li>
    <li>python313Packages.oslo-log.dist</li>
    <li>python313Packages.oslo-metrics</li>
    <li>python313Packages.oslo-metrics.dist</li>
    <li>python313Packages.oslo-serialization</li>
    <li>python313Packages.oslo-serialization.dist</li>
    <li>python313Packages.oslo-utils</li>
    <li>python313Packages.oslo-utils.dist</li>
    <li>python313Packages.osprofiler</li>
    <li>python313Packages.osprofiler.dist</li>
    <li>python313Packages.python-aodhclient</li>
    <li>python313Packages.python-aodhclient.dist</li>
    <li>python313Packages.python-cinderclient</li>
    <li>python313Packages.python-cinderclient.dist</li>
    <li>python313Packages.python-designateclient</li>
    <li>python313Packages.python-designateclient.dist</li>
    <li>python313Packages.python-keystoneclient</li>
    <li>python313Packages.python-keystoneclient.dist</li>
    <li>python313Packages.python-neutronclient</li>
    <li>python313Packages.python-neutronclient.dist</li>
    <li>python313Packages.python-novaclient</li>
    <li>python313Packages.python-novaclient.dist</li>
    <li>python313Packages.python-octaviaclient</li>
    <li>python313Packages.python-octaviaclient.dist</li>
    <li>swiftclient (python313Packages.python-swiftclient)</li>
    <li>swiftclient.dist (python313Packages.python-swiftclient.dist)</li>
    <li>troveclient (python313Packages.python-troveclient)</li>
    <li>troveclient.dist (python313Packages.python-troveclient.dist)</li>
    <li>watcherclient (python313Packages.python-watcherclient)</li>
    <li>watcherclient.dist (python313Packages.python-watcherclient.dist)</li>
    <li>python313Packages.python-zaqarclient</li>
    <li>python313Packages.python-zaqarclient.dist</li>
    <li>zunclient (python313Packages.python-zunclient)</li>
    <li>zunclient.dist (python313Packages.python-zunclient.dist)</li>
    <li>python313Packages.tempest</li>
    <li>python313Packages.tempest.dist</li>
    <li>qemu</li>
    <li>qemu-utils</li>
    <li>qemu-utils.debug</li>
    <li>qemu-utils.doc</li>
    <li>qemu.debug</li>
    <li>qemu.doc</li>
    <li>qemu.ga</li>
    <li>qemu_full</li>
    <li>qemu_full.debug</li>
    <li>qemu_full.doc</li>
    <li>qemu_full.ga</li>
    <li>qemu_kvm</li>
    <li>qemu_kvm.debug</li>
    <li>qemu_kvm.doc</li>
    <li>qemu_kvm.ga</li>
    <li>qemu_test</li>
    <li>qemu_test.debug</li>
    <li>qemu_test.doc</li>
    <li>qemu_test.ga</li>
    <li>qemu_xen</li>
    <li>qemu_xen.debug</li>
    <li>qemu_xen.doc</li>
    <li>qemu_xen.ga</li>
    <li>qtemu</li>
    <li>quickemu</li>
    <li>quickgui</li>
    <li>quickgui.debug</li>
    <li>quickgui.pubcache</li>
    <li>restool</li>
    <li>snagboot</li>
    <li>snagboot.dist</li>
    <li>solo5</li>
    <li>solo5.debug</li>
    <li>spike</li>
    <li>sunxi-tools</li>
    <li>tests.devShellTools.nixos</li>
    <li>tests.dotnet.final-attrs.check-output</li>
    <li>tests.dotnet.final-attrs.output-matches-const</li>
    <li>tests.dotnet.final-attrs.override-has-no-effect</li>
    <li>tests.dotnet.final-attrs.override-modifies-output</li>
    <li>tests.dotnet.structured-attrs.check-output</li>
    <li>tests.dotnet.use-dotnet-from-env.fallback</li>
    <li>tests.dotnet.use-dotnet-from-env.use-dotnet-path-env</li>
    <li>tests.dotnet.use-dotnet-from-env.use-dotnet-root-env</li>
    <li>tests.replaceVars.replaceVars.succeeds</li>
    <li>tests.replaceVars.replaceVars.succeeds-with-exemption</li>
    <li>tests.replaceVars.replaceVarsWith.succeeds</li>
    <li>tests.replaceVars.replaceVarsWith.succeeds-with-exemption</li>
    <li>tests.substitute.legacySingleArg</li>
    <li>tests.substitute.legacySingleReplace</li>
    <li>tests.substitute.legacyString</li>
    <li>tests.substitute.legacyVar</li>
    <li>tests.substitute.substitutions</li>
    <li>tests.testers.lycheeLinkCheck.network</li>
    <li>tests.testers.nixosTest-example</li>
    <li>tests.testers.runNixOSTest-example</li>
    <li>tests.testers.testEqualContents.emptyFileAndDir</li>
    <li>tests.testers.testEqualContents.equalDir</li>
    <li>tests.testers.testEqualContents.equalExe</li>
    <li>tests.testers.testEqualContents.fileDiff</li>
    <li>tests.testers.testEqualContents.fileMissing</li>
    <li>tests.testers.testEqualContents.nonExistentPath</li>
    <li>tests.testers.testEqualContents.unequalExe</li>
    <li>tests.testers.testEqualContents.unequalExeInDir</li>
    <li>tests.trivial-builders.references</li>
    <li>tests.trivial-builders.symlinkJoin.symlinkJoin</li>
    <li>tests.trivial-builders.symlinkJoin.symlinkJoin-strip-paths</li>
    <li>tests.trivial-builders.symlinkJoin.symlinkJoin-strip-paths-skip-missing</li>
    <li>tests.trivial-builders.symlinkJoin.symlinkJoin-strip-paths-skip-not-directories</li>
    <li>tests.writers.data.json</li>
    <li>tests.writers.data.toml</li>
    <li>tests.writers.data.yaml</li>
    <li>ubootQemuX86</li>
    <li>ubootQemuX86_64</li>
    <li>ubootTools</li>
    <li>ubootTools.man</li>
    <li>vagrant</li>
    <li>virt-v2v</li>
    <li>vmctl</li>
    <li>zpool-auto-expand-partitions</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `x86_64-linux`</summary>
<details>
<summary>python312Packages.libfdt</summary>
<pre>dtc -I dts -O dtb -o test_props.dtb /build/source/tests/test_props.dts:	PASS
Traceback (most recent call last):
  File "/build/source/tests/pylibfdt_tests.py", line 13, in <module>
    import libfdt
  File "/build/source/pylibfdt/libfdt.py", line 12, in <module>
    import _libfdt
ModuleNotFoundError: No module named '_libfdt'
stderr:
/build/source/tests/run_tests.sh: line 1107: arithmetic syntax error



Summary of Failures:

9/9 pylibfdt   FAIL            0.21s   exit status 2

Ok:                8   
Fail:              1   

Full log written to /build/source/build/meson-logs/testlog.txt</pre>
</details>
</details>


---
### `aarch64-linux` (sandbox = true)
<details>
  <summary>:x: 11 packages failed to build:</summary>
  <ul>
    <li>python312Packages.libfdt</li>
    <li>tests.devShellTools.nixos</li>
    <li>tests.testers.lycheeLinkCheck.network</li>
    <li>tests.testers.nixosTest-example</li>
    <li>tests.testers.runNixOSTest-example</li>
    <li>tests.trivial-builders.references</li>
    <li>ubootTuringRK1</li>
    <li>vagrant</li>
    <li>vmctl</li>
    <li>zepp-simulator</li>
    <li>zpool-auto-expand-partitions</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 263 packages built:</summary>
  <ul>
    <li>OVMFFull</li>
    <li>OVMFFull.fd</li>
    <li>alpine-make-vm-image</li>
    <li>argononed</li>
    <li>barbicanclient (python313Packages.python-barbicanclient)</li>
    <li>barbicanclient.dist (python313Packages.python-barbicanclient.dist)</li>
    <li>binwalk</li>
    <li>cloud-hypervisor</li>
    <li>cloud-hypervisor.debug</li>
    <li>cloud-init</li>
    <li>cloud-init.dist</li>
    <li>cloud-utils</li>
    <li>cloud-utils.guest</li>
    <li>colima</li>
    <li>cot (python313Packages.cot)</li>
    <li>cot.dist (python313Packages.cot.dist)</li>
    <li>culvert</li>
    <li>debos</li>
    <li>diffoscopeMinimal</li>
    <li>diffoscopeMinimal.dist</li>
    <li>diffoscopeMinimal.man</li>
    <li>dt-schema</li>
    <li>dt-schema.dist</li>
    <li>dtc</li>
    <li>fwts</li>
    <li>ghidra-extensions.ghidraninja-ghidra-scripts</li>
    <li>glanceclient (python313Packages.python-glanceclient)</li>
    <li>glanceclient.dist (python313Packages.python-glanceclient.dist)</li>
    <li>gnome-boxes</li>
    <li>goldboot</li>
    <li>heatclient (python313Packages.python-heatclient)</li>
    <li>heatclient.dist (python313Packages.python-heatclient.dist)</li>
    <li>hot-resize</li>
    <li>ironicclient (python313Packages.python-ironicclient)</li>
    <li>ironicclient.dist (python313Packages.python-ironicclient.dist)</li>
    <li>kata-runtime</li>
    <li>kvmtool</li>
    <li>libguestfs (python313Packages.guestfs)</li>
    <li>libguestfs.guestfsd (python313Packages.guestfs.guestfsd)</li>
    <li>lima</li>
    <li>lxd-lts</li>
    <li>magnumclient (python313Packages.python-magnumclient)</li>
    <li>magnumclient.dist (python313Packages.python-magnumclient.dist)</li>
    <li>manilaclient (python313Packages.python-manilaclient)</li>
    <li>manilaclient.dist (python313Packages.python-manilaclient.dist)</li>
    <li>mistralclient (python313Packages.python-mistralclient)</li>
    <li>mistralclient.dist (python313Packages.python-mistralclient.dist)</li>
    <li>nemu</li>
    <li>openstackclient (python313Packages.python-openstackclient)</li>
    <li>openstackclient-full</li>
    <li>openstackclient-full.dist</li>
    <li>openstackclient.dist (python313Packages.python-openstackclient.dist)</li>
    <li>out-of-tree</li>
    <li>python312Packages.cot</li>
    <li>python312Packages.cot.dist</li>
    <li>python312Packages.guestfs</li>
    <li>python312Packages.guestfs.guestfsd</li>
    <li>python312Packages.keystoneauth1</li>
    <li>python312Packages.keystoneauth1.dist</li>
    <li>python312Packages.openstacksdk</li>
    <li>python312Packages.openstacksdk.dist</li>
    <li>python312Packages.openstacksdk.man</li>
    <li>python312Packages.os-client-config</li>
    <li>python312Packages.os-client-config.dist</li>
    <li>python312Packages.osc-lib</li>
    <li>python312Packages.osc-lib.dist</li>
    <li>python312Packages.osc-placement</li>
    <li>python312Packages.osc-placement.dist</li>
    <li>python312Packages.oslo-concurrency</li>
    <li>python312Packages.oslo-concurrency.dist</li>
    <li>python312Packages.oslo-db</li>
    <li>python312Packages.oslo-db.dist</li>
    <li>python312Packages.oslo-log</li>
    <li>python312Packages.oslo-log.dist</li>
    <li>python312Packages.oslo-metrics</li>
    <li>python312Packages.oslo-metrics.dist</li>
    <li>python312Packages.oslo-serialization</li>
    <li>python312Packages.oslo-serialization.dist</li>
    <li>python312Packages.oslo-utils</li>
    <li>python312Packages.oslo-utils.dist</li>
    <li>python312Packages.osprofiler</li>
    <li>python312Packages.osprofiler.dist</li>
    <li>python312Packages.python-aodhclient</li>
    <li>python312Packages.python-aodhclient.dist</li>
    <li>python312Packages.python-barbicanclient</li>
    <li>python312Packages.python-barbicanclient.dist</li>
    <li>python312Packages.python-cinderclient</li>
    <li>python312Packages.python-cinderclient.dist</li>
    <li>python312Packages.python-designateclient</li>
    <li>python312Packages.python-designateclient.dist</li>
    <li>python312Packages.python-glanceclient</li>
    <li>python312Packages.python-glanceclient.dist</li>
    <li>python312Packages.python-heatclient</li>
    <li>python312Packages.python-heatclient.dist</li>
    <li>python312Packages.python-ironicclient</li>
    <li>python312Packages.python-ironicclient.dist</li>
    <li>python312Packages.python-keystoneclient</li>
    <li>python312Packages.python-keystoneclient.dist</li>
    <li>python312Packages.python-magnumclient</li>
    <li>python312Packages.python-magnumclient.dist</li>
    <li>python312Packages.python-manilaclient</li>
    <li>python312Packages.python-manilaclient.dist</li>
    <li>python312Packages.python-mistralclient</li>
    <li>python312Packages.python-mistralclient.dist</li>
    <li>python312Packages.python-neutronclient</li>
    <li>python312Packages.python-neutronclient.dist</li>
    <li>python312Packages.python-novaclient</li>
    <li>python312Packages.python-novaclient.dist</li>
    <li>python312Packages.python-octaviaclient</li>
    <li>python312Packages.python-octaviaclient.dist</li>
    <li>python312Packages.python-openstackclient</li>
    <li>python312Packages.python-openstackclient.dist</li>
    <li>python312Packages.python-swiftclient</li>
    <li>python312Packages.python-swiftclient.dist</li>
    <li>python312Packages.python-troveclient</li>
    <li>python312Packages.python-troveclient.dist</li>
    <li>python312Packages.python-watcherclient</li>
    <li>python312Packages.python-watcherclient.dist</li>
    <li>python312Packages.python-zaqarclient</li>
    <li>python312Packages.python-zaqarclient.dist</li>
    <li>python312Packages.python-zunclient</li>
    <li>python312Packages.python-zunclient.dist</li>
    <li>python312Packages.tempest</li>
    <li>python312Packages.tempest.dist</li>
    <li>python313Packages.keystoneauth1</li>
    <li>python313Packages.keystoneauth1.dist</li>
    <li>python313Packages.libfdt</li>
    <li>python313Packages.openstacksdk</li>
    <li>python313Packages.openstacksdk.dist</li>
    <li>python313Packages.openstacksdk.man</li>
    <li>python313Packages.os-client-config</li>
    <li>python313Packages.os-client-config.dist</li>
    <li>python313Packages.osc-lib</li>
    <li>python313Packages.osc-lib.dist</li>
    <li>python313Packages.osc-placement</li>
    <li>python313Packages.osc-placement.dist</li>
    <li>python313Packages.oslo-concurrency</li>
    <li>python313Packages.oslo-concurrency.dist</li>
    <li>python313Packages.oslo-db</li>
    <li>python313Packages.oslo-db.dist</li>
    <li>python313Packages.oslo-log</li>
    <li>python313Packages.oslo-log.dist</li>
    <li>python313Packages.oslo-metrics</li>
    <li>python313Packages.oslo-metrics.dist</li>
    <li>python313Packages.oslo-serialization</li>
    <li>python313Packages.oslo-serialization.dist</li>
    <li>python313Packages.oslo-utils</li>
    <li>python313Packages.oslo-utils.dist</li>
    <li>python313Packages.osprofiler</li>
    <li>python313Packages.osprofiler.dist</li>
    <li>python313Packages.python-aodhclient</li>
    <li>python313Packages.python-aodhclient.dist</li>
    <li>python313Packages.python-cinderclient</li>
    <li>python313Packages.python-cinderclient.dist</li>
    <li>python313Packages.python-designateclient</li>
    <li>python313Packages.python-designateclient.dist</li>
    <li>python313Packages.python-keystoneclient</li>
    <li>python313Packages.python-keystoneclient.dist</li>
    <li>python313Packages.python-neutronclient</li>
    <li>python313Packages.python-neutronclient.dist</li>
    <li>python313Packages.python-novaclient</li>
    <li>python313Packages.python-novaclient.dist</li>
    <li>python313Packages.python-octaviaclient</li>
    <li>python313Packages.python-octaviaclient.dist</li>
    <li>swiftclient (python313Packages.python-swiftclient)</li>
    <li>swiftclient.dist (python313Packages.python-swiftclient.dist)</li>
    <li>troveclient (python313Packages.python-troveclient)</li>
    <li>troveclient.dist (python313Packages.python-troveclient.dist)</li>
    <li>watcherclient (python313Packages.python-watcherclient)</li>
    <li>watcherclient.dist (python313Packages.python-watcherclient.dist)</li>
    <li>python313Packages.python-zaqarclient</li>
    <li>python313Packages.python-zaqarclient.dist</li>
    <li>zunclient (python313Packages.python-zunclient)</li>
    <li>zunclient.dist (python313Packages.python-zunclient.dist)</li>
    <li>python313Packages.tempest</li>
    <li>python313Packages.tempest.dist</li>
    <li>qemu</li>
    <li>qemu-utils</li>
    <li>qemu-utils.doc</li>
    <li>qemu.doc</li>
    <li>qemu.ga</li>
    <li>qemu_full</li>
    <li>qemu_full.doc</li>
    <li>qemu_full.ga</li>
    <li>qemu_kvm</li>
    <li>qemu_kvm.doc</li>
    <li>qemu_kvm.ga</li>
    <li>qemu_test</li>
    <li>qemu_test.doc</li>
    <li>qemu_test.ga</li>
    <li>qtemu</li>
    <li>quickemu</li>
    <li>restool</li>
    <li>rkboot</li>
    <li>snagboot</li>
    <li>snagboot.dist</li>
    <li>spike</li>
    <li>sunxi-tools</li>
    <li>tests.dotnet.final-attrs.check-output</li>
    <li>tests.dotnet.final-attrs.output-matches-const</li>
    <li>tests.dotnet.final-attrs.override-has-no-effect</li>
    <li>tests.dotnet.final-attrs.override-modifies-output</li>
    <li>tests.dotnet.structured-attrs.check-output</li>
    <li>tests.dotnet.use-dotnet-from-env.fallback</li>
    <li>tests.dotnet.use-dotnet-from-env.use-dotnet-path-env</li>
    <li>tests.dotnet.use-dotnet-from-env.use-dotnet-root-env</li>
    <li>tests.replaceVars.replaceVars.succeeds</li>
    <li>tests.replaceVars.replaceVars.succeeds-with-exemption</li>
    <li>tests.replaceVars.replaceVarsWith.succeeds</li>
    <li>tests.replaceVars.replaceVarsWith.succeeds-with-exemption</li>
    <li>tests.substitute.legacySingleArg</li>
    <li>tests.substitute.legacySingleReplace</li>
    <li>tests.substitute.legacyString</li>
    <li>tests.substitute.legacyVar</li>
    <li>tests.substitute.substitutions</li>
    <li>tests.testers.testEqualContents.emptyFileAndDir</li>
    <li>tests.testers.testEqualContents.equalDir</li>
    <li>tests.testers.testEqualContents.equalExe</li>
    <li>tests.testers.testEqualContents.fileDiff</li>
    <li>tests.testers.testEqualContents.fileMissing</li>
    <li>tests.testers.testEqualContents.nonExistentPath</li>
    <li>tests.testers.testEqualContents.unequalExe</li>
    <li>tests.testers.testEqualContents.unequalExeInDir</li>
    <li>tests.trivial-builders.symlinkJoin.symlinkJoin</li>
    <li>tests.trivial-builders.symlinkJoin.symlinkJoin-strip-paths</li>
    <li>tests.trivial-builders.symlinkJoin.symlinkJoin-strip-paths-skip-missing</li>
    <li>tests.trivial-builders.symlinkJoin.symlinkJoin-strip-paths-skip-not-directories</li>
    <li>tests.writers.data.json</li>
    <li>tests.writers.data.toml</li>
    <li>tests.writers.data.yaml</li>
    <li>ubootBananaPim64</li>
    <li>ubootCM3588NAS</li>
    <li>ubootNanoPCT4</li>
    <li>ubootNanoPCT6</li>
    <li>ubootOdroidC2</li>
    <li>ubootOlimexA64Olinuxino</li>
    <li>ubootOlimexA64Teres1</li>
    <li>ubootOrangePi3</li>
    <li>ubootOrangePi3B</li>
    <li>ubootOrangePi5</li>
    <li>ubootOrangePi5Plus</li>
    <li>ubootOrangePiZero2</li>
    <li>ubootOrangePiZero3</li>
    <li>ubootOrangePiZeroPlus2H5</li>
    <li>ubootPine64</li>
    <li>ubootPine64LTS</li>
    <li>ubootPinebook</li>
    <li>ubootPinebookPro</li>
    <li>ubootQemuAarch64</li>
    <li>ubootQuartz64B</li>
    <li>ubootROCPCRK3399</li>
    <li>ubootRaspberryPi3_64bit</li>
    <li>ubootRaspberryPi4_64bit</li>
    <li>ubootRock4CPlus</li>
    <li>ubootRock5ModelB</li>
    <li>ubootRock64</li>
    <li>ubootRock64v2</li>
    <li>ubootRockPi4</li>
    <li>ubootRockPiE</li>
    <li>ubootRockPro64</li>
    <li>ubootSopine</li>
    <li>ubootTools</li>
    <li>ubootTools.man</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `aarch64-linux`</summary>
<details>
<summary>python312Packages.libfdt</summary>
<pre>dtc -I dts -O dtb -o test_props.dtb /build/source/tests/test_props.dts:	PASS
Traceback (most recent call last):
  File "/build/source/tests/pylibfdt_tests.py", line 13, in <module>
    import libfdt
  File "/build/source/pylibfdt/libfdt.py", line 12, in <module>
    import _libfdt
ModuleNotFoundError: No module named '_libfdt'
stderr:
/build/source/tests/run_tests.sh: line 1107: arithmetic syntax error



Summary of Failures:

9/9 pylibfdt   FAIL            0.08s   exit status 2

Ok:                8   
Fail:              1   

Full log written to /build/source/build/meson-logs/testlog.txt</pre>
</details>
<details>
<summary>ubootTuringRK1</summary>
<pre>  CC      lib/sha1.o
  CC      lib/efi_loader/efi_string.o
  CC      lib/sha256.o
  CC      lib/gunzip.o
  CC      lib/efi_loader/efi_unicode_collation.o
  CC      lib/lz4_wrapper.o
  CC      lib/efi_loader/efi_var_common.o
  CC      lib/fdtdec.o
  CC      lib/qsort.o
  CC      lib/efi_loader/efi_var_mem.o
  CC      lib/hashtable.o
  CC      lib/errno.o
  CC      lib/efi_loader/efi_variable.o
  CC      lib/efi_loader/efi_var_file.o
  CC      lib/efi_loader/efi_watchdog.o
  CC      lib/display_options.o
  CC      lib/efi_loader/efi_disk.o
  CC      lib/efi_loader/efi_net.o
  CC      lib/crc32.o
  CC      lib/ctype.o</pre>
</details>
</details>


---
### `x86_64-darwin` (sandbox = true)
<details>
  <summary>:x: 226 packages failed to build:</summary>
  <ul>
    <li>OVMFFull</li>
    <li>OVMFFull.fd</li>
    <li>appvm</li>
    <li>barbicanclient (python313Packages.python-barbicanclient)</li>
    <li>barbicanclient.dist (python313Packages.python-barbicanclient.dist)</li>
    <li>binwalk</li>
    <li>cloud-utils</li>
    <li>cloud-utils.guest</li>
    <li>colima</li>
    <li>cot (python313Packages.cot)</li>
    <li>cot.dist (python313Packages.cot.dist)</li>
    <li>darwin.linux-builder (darwin.linux-builder-x86_64)</li>
    <li>diffoscope</li>
    <li>diffoscope.dist</li>
    <li>diffoscope.man</li>
    <li>diffoscopeMinimal</li>
    <li>diffoscopeMinimal.dist</li>
    <li>diffoscopeMinimal.man</li>
    <li>dtc</li>
    <li>fdroidserver</li>
    <li>fdroidserver.dist</li>
    <li>ghidra-extensions.ghidraninja-ghidra-scripts</li>
    <li>glanceclient (python313Packages.python-glanceclient)</li>
    <li>glanceclient.dist (python313Packages.python-glanceclient.dist)</li>
    <li>goldboot</li>
    <li>heatclient (python313Packages.python-heatclient)</li>
    <li>heatclient.dist (python313Packages.python-heatclient.dist)</li>
    <li>ironicclient (python313Packages.python-ironicclient)</li>
    <li>ironicclient.dist (python313Packages.python-ironicclient.dist)</li>
    <li>libvirt</li>
    <li>libvirt-glib</li>
    <li>libvirt-glib.dev</li>
    <li>libvirt-glib.devdoc</li>
    <li>lima</li>
    <li>magnumclient (python313Packages.python-magnumclient)</li>
    <li>magnumclient.dist (python313Packages.python-magnumclient.dist)</li>
    <li>manilaclient (python313Packages.python-manilaclient)</li>
    <li>manilaclient.dist (python313Packages.python-manilaclient.dist)</li>
    <li>mgmt</li>
    <li>mistralclient (python313Packages.python-mistralclient)</li>
    <li>mistralclient.dist (python313Packages.python-mistralclient.dist)</li>
    <li>openstackclient (python313Packages.python-openstackclient)</li>
    <li>openstackclient-full</li>
    <li>openstackclient-full.dist</li>
    <li>openstackclient.dist (python313Packages.python-openstackclient.dist)</li>
    <li>out-of-tree</li>
    <li>perl538Packages.SysVirt</li>
    <li>perl538Packages.SysVirt.devdoc</li>
    <li>perl540Packages.SysVirt</li>
    <li>perl540Packages.SysVirt.devdoc</li>
    <li>prometheus-libvirt-exporter</li>
    <li>python312Packages.cot</li>
    <li>python312Packages.cot.dist</li>
    <li>python312Packages.keystoneauth1</li>
    <li>python312Packages.keystoneauth1.dist</li>
    <li>python312Packages.libfdt</li>
    <li>python312Packages.libvirt</li>
    <li>python312Packages.libvirt.dist</li>
    <li>python312Packages.openstacksdk</li>
    <li>python312Packages.openstacksdk.dist</li>
    <li>python312Packages.openstacksdk.man</li>
    <li>python312Packages.os-client-config</li>
    <li>python312Packages.os-client-config.dist</li>
    <li>python312Packages.osc-lib</li>
    <li>python312Packages.osc-lib.dist</li>
    <li>python312Packages.osc-placement</li>
    <li>python312Packages.osc-placement.dist</li>
    <li>python312Packages.oslo-concurrency</li>
    <li>python312Packages.oslo-concurrency.dist</li>
    <li>python312Packages.oslo-db</li>
    <li>python312Packages.oslo-db.dist</li>
    <li>python312Packages.oslo-log</li>
    <li>python312Packages.oslo-log.dist</li>
    <li>python312Packages.oslo-metrics</li>
    <li>python312Packages.oslo-metrics.dist</li>
    <li>python312Packages.oslo-serialization</li>
    <li>python312Packages.oslo-serialization.dist</li>
    <li>python312Packages.oslo-utils</li>
    <li>python312Packages.oslo-utils.dist</li>
    <li>python312Packages.osprofiler</li>
    <li>python312Packages.osprofiler.dist</li>
    <li>python312Packages.python-aodhclient</li>
    <li>python312Packages.python-aodhclient.dist</li>
    <li>python312Packages.python-barbicanclient</li>
    <li>python312Packages.python-barbicanclient.dist</li>
    <li>python312Packages.python-cinderclient</li>
    <li>python312Packages.python-cinderclient.dist</li>
    <li>python312Packages.python-designateclient</li>
    <li>python312Packages.python-designateclient.dist</li>
    <li>python312Packages.python-glanceclient</li>
    <li>python312Packages.python-glanceclient.dist</li>
    <li>python312Packages.python-heatclient</li>
    <li>python312Packages.python-heatclient.dist</li>
    <li>python312Packages.python-ironicclient</li>
    <li>python312Packages.python-ironicclient.dist</li>
    <li>python312Packages.python-keystoneclient</li>
    <li>python312Packages.python-keystoneclient.dist</li>
    <li>python312Packages.python-magnumclient</li>
    <li>python312Packages.python-magnumclient.dist</li>
    <li>python312Packages.python-manilaclient</li>
    <li>python312Packages.python-manilaclient.dist</li>
    <li>python312Packages.python-mistralclient</li>
    <li>python312Packages.python-mistralclient.dist</li>
    <li>python312Packages.python-neutronclient</li>
    <li>python312Packages.python-neutronclient.dist</li>
    <li>python312Packages.python-novaclient</li>
    <li>python312Packages.python-novaclient.dist</li>
    <li>python312Packages.python-octaviaclient</li>
    <li>python312Packages.python-octaviaclient.dist</li>
    <li>python312Packages.python-openstackclient</li>
    <li>python312Packages.python-openstackclient.dist</li>
    <li>python312Packages.python-swiftclient</li>
    <li>python312Packages.python-swiftclient.dist</li>
    <li>python312Packages.python-troveclient</li>
    <li>python312Packages.python-troveclient.dist</li>
    <li>python312Packages.python-watcherclient</li>
    <li>python312Packages.python-watcherclient.dist</li>
    <li>python312Packages.python-zaqarclient</li>
    <li>python312Packages.python-zaqarclient.dist</li>
    <li>python312Packages.python-zunclient</li>
    <li>python312Packages.python-zunclient.dist</li>
    <li>python312Packages.tempest</li>
    <li>python312Packages.tempest.dist</li>
    <li>python313Packages.keystoneauth1</li>
    <li>python313Packages.keystoneauth1.dist</li>
    <li>python313Packages.libfdt</li>
    <li>python313Packages.libvirt</li>
    <li>python313Packages.libvirt.dist</li>
    <li>python313Packages.openstacksdk</li>
    <li>python313Packages.openstacksdk.dist</li>
    <li>python313Packages.openstacksdk.man</li>
    <li>python313Packages.os-client-config</li>
    <li>python313Packages.os-client-config.dist</li>
    <li>python313Packages.osc-lib</li>
    <li>python313Packages.osc-lib.dist</li>
    <li>python313Packages.osc-placement</li>
    <li>python313Packages.osc-placement.dist</li>
    <li>python313Packages.oslo-concurrency</li>
    <li>python313Packages.oslo-concurrency.dist</li>
    <li>python313Packages.oslo-db</li>
    <li>python313Packages.oslo-db.dist</li>
    <li>python313Packages.oslo-log</li>
    <li>python313Packages.oslo-log.dist</li>
    <li>python313Packages.oslo-metrics</li>
    <li>python313Packages.oslo-metrics.dist</li>
    <li>python313Packages.oslo-serialization</li>
    <li>python313Packages.oslo-serialization.dist</li>
    <li>python313Packages.oslo-utils</li>
    <li>python313Packages.oslo-utils.dist</li>
    <li>python313Packages.osprofiler</li>
    <li>python313Packages.osprofiler.dist</li>
    <li>python313Packages.python-aodhclient</li>
    <li>python313Packages.python-aodhclient.dist</li>
    <li>python313Packages.python-cinderclient</li>
    <li>python313Packages.python-cinderclient.dist</li>
    <li>python313Packages.python-designateclient</li>
    <li>python313Packages.python-designateclient.dist</li>
    <li>python313Packages.python-keystoneclient</li>
    <li>python313Packages.python-keystoneclient.dist</li>
    <li>python313Packages.python-neutronclient</li>
    <li>python313Packages.python-neutronclient.dist</li>
    <li>python313Packages.python-novaclient</li>
    <li>python313Packages.python-novaclient.dist</li>
    <li>python313Packages.python-octaviaclient</li>
    <li>python313Packages.python-octaviaclient.dist</li>
    <li>swiftclient (python313Packages.python-swiftclient)</li>
    <li>swiftclient.dist (python313Packages.python-swiftclient.dist)</li>
    <li>troveclient (python313Packages.python-troveclient)</li>
    <li>troveclient.dist (python313Packages.python-troveclient.dist)</li>
    <li>watcherclient (python313Packages.python-watcherclient)</li>
    <li>watcherclient.dist (python313Packages.python-watcherclient.dist)</li>
    <li>python313Packages.python-zaqarclient</li>
    <li>python313Packages.python-zaqarclient.dist</li>
    <li>zunclient (python313Packages.python-zunclient)</li>
    <li>zunclient.dist (python313Packages.python-zunclient.dist)</li>
    <li>python313Packages.tempest</li>
    <li>python313Packages.tempest.dist</li>
    <li>qemu</li>
    <li>qemu-utils</li>
    <li>qemu-utils.doc</li>
    <li>qemu.doc</li>
    <li>qemu_full</li>
    <li>qemu_full.doc</li>
    <li>qemu_kvm</li>
    <li>qemu_kvm.doc</li>
    <li>qemu_test</li>
    <li>qemu_test.doc</li>
    <li>quickemu</li>
    <li>rubyPackages.ruby-libvirt (rubyPackages_3_3.ruby-libvirt)</li>
    <li>rubyPackages_3_1.ruby-libvirt</li>
    <li>rubyPackages_3_2.ruby-libvirt</li>
    <li>rubyPackages_3_4.ruby-libvirt</li>
    <li>tests.dotnet.final-attrs.check-output</li>
    <li>tests.dotnet.final-attrs.output-matches-const</li>
    <li>tests.dotnet.final-attrs.override-has-no-effect</li>
    <li>tests.dotnet.final-attrs.override-modifies-output</li>
    <li>tests.dotnet.structured-attrs.check-output</li>
    <li>tests.dotnet.use-dotnet-from-env.fallback</li>
    <li>tests.dotnet.use-dotnet-from-env.use-dotnet-path-env</li>
    <li>tests.dotnet.use-dotnet-from-env.use-dotnet-root-env</li>
    <li>tests.replaceVars.replaceVars.succeeds</li>
    <li>tests.replaceVars.replaceVars.succeeds-with-exemption</li>
    <li>tests.replaceVars.replaceVarsWith.succeeds</li>
    <li>tests.replaceVars.replaceVarsWith.succeeds-with-exemption</li>
    <li>tests.substitute.legacySingleArg</li>
    <li>tests.substitute.legacySingleReplace</li>
    <li>tests.substitute.legacyString</li>
    <li>tests.substitute.legacyVar</li>
    <li>tests.substitute.substitutions</li>
    <li>tests.testers.testEqualContents.emptyFileAndDir</li>
    <li>tests.testers.testEqualContents.equalDir</li>
    <li>tests.testers.testEqualContents.equalExe</li>
    <li>tests.testers.testEqualContents.fileDiff</li>
    <li>tests.testers.testEqualContents.fileMissing</li>
    <li>tests.testers.testEqualContents.nonExistentPath</li>
    <li>tests.testers.testEqualContents.unequalExe</li>
    <li>tests.testers.testEqualContents.unequalExeInDir</li>
    <li>tests.trivial-builders.symlinkJoin.symlinkJoin</li>
    <li>tests.trivial-builders.symlinkJoin.symlinkJoin-strip-paths</li>
    <li>tests.trivial-builders.symlinkJoin.symlinkJoin-strip-paths-skip-missing</li>
    <li>tests.trivial-builders.symlinkJoin.symlinkJoin-strip-paths-skip-not-directories</li>
    <li>tests.writers.data.json</li>
    <li>tests.writers.data.toml</li>
    <li>tests.writers.data.yaml</li>
    <li>virt-manager</li>
    <li>virt-viewer</li>
  </ul>
</details>


---
### `aarch64-darwin` (sandbox = true)
<details>
  <summary>:x: 217 packages failed to build:</summary>
  <ul>
    <li>appvm</li>
    <li>barbicanclient (python313Packages.python-barbicanclient)</li>
    <li>barbicanclient.dist (python313Packages.python-barbicanclient.dist)</li>
    <li>binwalk</li>
    <li>cloud-utils</li>
    <li>cloud-utils.guest</li>
    <li>colima</li>
    <li>cot (python313Packages.cot)</li>
    <li>cot.dist (python313Packages.cot.dist)</li>
    <li>darwin.linux-builder</li>
    <li>darwin.linux-builder-x86_64</li>
    <li>diffoscope</li>
    <li>diffoscope.dist</li>
    <li>diffoscope.man</li>
    <li>diffoscopeMinimal</li>
    <li>diffoscopeMinimal.dist</li>
    <li>diffoscopeMinimal.man</li>
    <li>fdroidserver</li>
    <li>fdroidserver.dist</li>
    <li>ghidra-extensions.ghidraninja-ghidra-scripts</li>
    <li>glanceclient (python313Packages.python-glanceclient)</li>
    <li>glanceclient.dist (python313Packages.python-glanceclient.dist)</li>
    <li>heatclient (python313Packages.python-heatclient)</li>
    <li>heatclient.dist (python313Packages.python-heatclient.dist)</li>
    <li>ironicclient (python313Packages.python-ironicclient)</li>
    <li>ironicclient.dist (python313Packages.python-ironicclient.dist)</li>
    <li>libvirt</li>
    <li>libvirt-glib</li>
    <li>libvirt-glib.dev</li>
    <li>libvirt-glib.devdoc</li>
    <li>lima</li>
    <li>magnumclient (python313Packages.python-magnumclient)</li>
    <li>magnumclient.dist (python313Packages.python-magnumclient.dist)</li>
    <li>manilaclient (python313Packages.python-manilaclient)</li>
    <li>manilaclient.dist (python313Packages.python-manilaclient.dist)</li>
    <li>mgmt</li>
    <li>mistralclient (python313Packages.python-mistralclient)</li>
    <li>mistralclient.dist (python313Packages.python-mistralclient.dist)</li>
    <li>openstackclient (python313Packages.python-openstackclient)</li>
    <li>openstackclient-full</li>
    <li>openstackclient-full.dist</li>
    <li>openstackclient.dist (python313Packages.python-openstackclient.dist)</li>
    <li>out-of-tree</li>
    <li>perl538Packages.SysVirt</li>
    <li>perl538Packages.SysVirt.devdoc</li>
    <li>perl540Packages.SysVirt</li>
    <li>perl540Packages.SysVirt.devdoc</li>
    <li>podman-bootc</li>
    <li>prometheus-libvirt-exporter</li>
    <li>python312Packages.cot</li>
    <li>python312Packages.cot.dist</li>
    <li>python312Packages.keystoneauth1</li>
    <li>python312Packages.keystoneauth1.dist</li>
    <li>python312Packages.libvirt</li>
    <li>python312Packages.libvirt.dist</li>
    <li>python312Packages.openstacksdk</li>
    <li>python312Packages.openstacksdk.dist</li>
    <li>python312Packages.openstacksdk.man</li>
    <li>python312Packages.os-client-config</li>
    <li>python312Packages.os-client-config.dist</li>
    <li>python312Packages.osc-lib</li>
    <li>python312Packages.osc-lib.dist</li>
    <li>python312Packages.osc-placement</li>
    <li>python312Packages.osc-placement.dist</li>
    <li>python312Packages.oslo-concurrency</li>
    <li>python312Packages.oslo-concurrency.dist</li>
    <li>python312Packages.oslo-db</li>
    <li>python312Packages.oslo-db.dist</li>
    <li>python312Packages.oslo-log</li>
    <li>python312Packages.oslo-log.dist</li>
    <li>python312Packages.oslo-metrics</li>
    <li>python312Packages.oslo-metrics.dist</li>
    <li>python312Packages.oslo-serialization</li>
    <li>python312Packages.oslo-serialization.dist</li>
    <li>python312Packages.oslo-utils</li>
    <li>python312Packages.oslo-utils.dist</li>
    <li>python312Packages.osprofiler</li>
    <li>python312Packages.osprofiler.dist</li>
    <li>python312Packages.python-aodhclient</li>
    <li>python312Packages.python-aodhclient.dist</li>
    <li>python312Packages.python-barbicanclient</li>
    <li>python312Packages.python-barbicanclient.dist</li>
    <li>python312Packages.python-cinderclient</li>
    <li>python312Packages.python-cinderclient.dist</li>
    <li>python312Packages.python-designateclient</li>
    <li>python312Packages.python-designateclient.dist</li>
    <li>python312Packages.python-glanceclient</li>
    <li>python312Packages.python-glanceclient.dist</li>
    <li>python312Packages.python-heatclient</li>
    <li>python312Packages.python-heatclient.dist</li>
    <li>python312Packages.python-ironicclient</li>
    <li>python312Packages.python-ironicclient.dist</li>
    <li>python312Packages.python-keystoneclient</li>
    <li>python312Packages.python-keystoneclient.dist</li>
    <li>python312Packages.python-magnumclient</li>
    <li>python312Packages.python-magnumclient.dist</li>
    <li>python312Packages.python-manilaclient</li>
    <li>python312Packages.python-manilaclient.dist</li>
    <li>python312Packages.python-mistralclient</li>
    <li>python312Packages.python-mistralclient.dist</li>
    <li>python312Packages.python-neutronclient</li>
    <li>python312Packages.python-neutronclient.dist</li>
    <li>python312Packages.python-novaclient</li>
    <li>python312Packages.python-novaclient.dist</li>
    <li>python312Packages.python-octaviaclient</li>
    <li>python312Packages.python-octaviaclient.dist</li>
    <li>python312Packages.python-openstackclient</li>
    <li>python312Packages.python-openstackclient.dist</li>
    <li>python312Packages.python-swiftclient</li>
    <li>python312Packages.python-swiftclient.dist</li>
    <li>python312Packages.python-troveclient</li>
    <li>python312Packages.python-troveclient.dist</li>
    <li>python312Packages.python-watcherclient</li>
    <li>python312Packages.python-watcherclient.dist</li>
    <li>python312Packages.python-zaqarclient</li>
    <li>python312Packages.python-zaqarclient.dist</li>
    <li>python312Packages.python-zunclient</li>
    <li>python312Packages.python-zunclient.dist</li>
    <li>python312Packages.tempest</li>
    <li>python312Packages.tempest.dist</li>
    <li>python313Packages.keystoneauth1</li>
    <li>python313Packages.keystoneauth1.dist</li>
    <li>python313Packages.libvirt</li>
    <li>python313Packages.libvirt.dist</li>
    <li>python313Packages.openstacksdk</li>
    <li>python313Packages.openstacksdk.dist</li>
    <li>python313Packages.openstacksdk.man</li>
    <li>python313Packages.os-client-config</li>
    <li>python313Packages.os-client-config.dist</li>
    <li>python313Packages.osc-lib</li>
    <li>python313Packages.osc-lib.dist</li>
    <li>python313Packages.osc-placement</li>
    <li>python313Packages.osc-placement.dist</li>
    <li>python313Packages.oslo-concurrency</li>
    <li>python313Packages.oslo-concurrency.dist</li>
    <li>python313Packages.oslo-db</li>
    <li>python313Packages.oslo-db.dist</li>
    <li>python313Packages.oslo-log</li>
    <li>python313Packages.oslo-log.dist</li>
    <li>python313Packages.oslo-metrics</li>
    <li>python313Packages.oslo-metrics.dist</li>
    <li>python313Packages.oslo-serialization</li>
    <li>python313Packages.oslo-serialization.dist</li>
    <li>python313Packages.oslo-utils</li>
    <li>python313Packages.oslo-utils.dist</li>
    <li>python313Packages.osprofiler</li>
    <li>python313Packages.osprofiler.dist</li>
    <li>python313Packages.python-aodhclient</li>
    <li>python313Packages.python-aodhclient.dist</li>
    <li>python313Packages.python-cinderclient</li>
    <li>python313Packages.python-cinderclient.dist</li>
    <li>python313Packages.python-designateclient</li>
    <li>python313Packages.python-designateclient.dist</li>
    <li>python313Packages.python-keystoneclient</li>
    <li>python313Packages.python-keystoneclient.dist</li>
    <li>python313Packages.python-neutronclient</li>
    <li>python313Packages.python-neutronclient.dist</li>
    <li>python313Packages.python-novaclient</li>
    <li>python313Packages.python-novaclient.dist</li>
    <li>python313Packages.python-octaviaclient</li>
    <li>python313Packages.python-octaviaclient.dist</li>
    <li>swiftclient (python313Packages.python-swiftclient)</li>
    <li>swiftclient.dist (python313Packages.python-swiftclient.dist)</li>
    <li>troveclient (python313Packages.python-troveclient)</li>
    <li>troveclient.dist (python313Packages.python-troveclient.dist)</li>
    <li>watcherclient (python313Packages.python-watcherclient)</li>
    <li>watcherclient.dist (python313Packages.python-watcherclient.dist)</li>
    <li>python313Packages.python-zaqarclient</li>
    <li>python313Packages.python-zaqarclient.dist</li>
    <li>zunclient (python313Packages.python-zunclient)</li>
    <li>zunclient.dist (python313Packages.python-zunclient.dist)</li>
    <li>python313Packages.tempest</li>
    <li>python313Packages.tempest.dist</li>
    <li>qemu-utils</li>
    <li>qemu-utils.doc</li>
    <li>qemu_kvm</li>
    <li>qemu_kvm.doc</li>
    <li>qemu_test</li>
    <li>qemu_test.doc</li>
    <li>rubyPackages.ruby-libvirt (rubyPackages_3_3.ruby-libvirt)</li>
    <li>rubyPackages_3_1.ruby-libvirt</li>
    <li>rubyPackages_3_2.ruby-libvirt</li>
    <li>rubyPackages_3_4.ruby-libvirt</li>
    <li>tests.dotnet.final-attrs.check-output</li>
    <li>tests.dotnet.final-attrs.output-matches-const</li>
    <li>tests.dotnet.final-attrs.override-has-no-effect</li>
    <li>tests.dotnet.final-attrs.override-modifies-output</li>
    <li>tests.dotnet.structured-attrs.check-output</li>
    <li>tests.dotnet.use-dotnet-from-env.fallback</li>
    <li>tests.dotnet.use-dotnet-from-env.use-dotnet-path-env</li>
    <li>tests.dotnet.use-dotnet-from-env.use-dotnet-root-env</li>
    <li>tests.replaceVars.replaceVars.succeeds</li>
    <li>tests.replaceVars.replaceVars.succeeds-with-exemption</li>
    <li>tests.replaceVars.replaceVarsWith.succeeds</li>
    <li>tests.replaceVars.replaceVarsWith.succeeds-with-exemption</li>
    <li>tests.substitute.legacySingleArg</li>
    <li>tests.substitute.legacySingleReplace</li>
    <li>tests.substitute.legacyString</li>
    <li>tests.substitute.legacyVar</li>
    <li>tests.substitute.substitutions</li>
    <li>tests.testers.testEqualContents.emptyFileAndDir</li>
    <li>tests.testers.testEqualContents.equalDir</li>
    <li>tests.testers.testEqualContents.equalExe</li>
    <li>tests.testers.testEqualContents.fileDiff</li>
    <li>tests.testers.testEqualContents.fileMissing</li>
    <li>tests.testers.testEqualContents.nonExistentPath</li>
    <li>tests.testers.testEqualContents.unequalExe</li>
    <li>tests.testers.testEqualContents.unequalExeInDir</li>
    <li>tests.trivial-builders.symlinkJoin.symlinkJoin</li>
    <li>tests.trivial-builders.symlinkJoin.symlinkJoin-strip-paths</li>
    <li>tests.trivial-builders.symlinkJoin.symlinkJoin-strip-paths-skip-missing</li>
    <li>tests.trivial-builders.symlinkJoin.symlinkJoin-strip-paths-skip-not-directories</li>
    <li>tests.writers.data.json</li>
    <li>tests.writers.data.toml</li>
    <li>tests.writers.data.yaml</li>
    <li>virt-manager</li>
    <li>virt-viewer</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 7 packages built:</summary>
  <ul>
    <li>dtc</li>
    <li>python312Packages.libfdt</li>
    <li>python313Packages.libfdt</li>
    <li>qemu</li>
    <li>qemu.doc</li>
    <li>qemu_full</li>
    <li>qemu_full.doc</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `aarch64-darwin`</summary>
<details>
<summary>binwalk</summary>
<pre>Using versionCheckHook
Running phase: unpackPhase
@nix { "action": "setPhase", "phase": "unpackPhase" }
unpacking source archive /nix/store/sz17s3c58ci5r9p8js1q8rhyslc2l6ga-source
source root is source
Executing cargoSetupPostUnpackHook</pre>
</details>
<details>
<summary>diffoscopeMinimal</summary>
<pre>tests/comparators/test_docx.py::test_no_differences PASSED               [ 15%]
tests/comparators/test_docx.py::test_diff SKIPPED (requires docx2txt)    [ 15%]
tests/comparators/test_docx.py::test_compare_non_existing SKIPPED        [ 16%]
tests/comparators/test_dtb.py::test_identification PASSED                [ 16%]
tests/comparators/test_dtb.py::test_no_differences PASSED                [ 16%]
tests/comparators/test_dtb.py::test_diff PASSED                          [ 16%]
tests/comparators/test_dtb.py::test_compare_non_existing PASSED          [ 16%]
tests/comparators/test_elf.py::test_obj_identification PASSED            [ 16%]
tests/comparators/test_elf.py::test_obj_no_differences PASSED            [ 16%]
tests/comparators/test_elf.py::test_obj_compare_non_existing PASSED      [ 16%]
tests/comparators/test_elf.py::test_diff PASSED                          [ 17%]
tests/comparators/test_elf.py::test_lib_identification PASSED            [ 17%]
tests/comparators/test_elf.py::test_lib_no_differences PASSED            [ 17%]
tests/comparators/test_elf.py::test_lib_differences PASSED               [ 17%]
tests/comparators/test_elf.py::test_lib_compare_non_existing PASSED      [ 17%]
tests/comparators/test_elf.py::test_differences_with_dbgsym PASSED       [ 17%]
tests/comparators/test_elf.py::test_original_gnu_debuglink PASSED        [ 17%]
tests/comparators/test_elf.py::test_ignore_readelf_errors1_identify PASSED [ 18%]
tests/comparators/test_elf.py::test_ignore_readelf_errors2_identify PASSED [ 18%]
tests/comparators/test_elf.py::test_ignore_readelf_errors </pre>
</details>
</details>

kirelagin commented:
Yeah, it is failing like this on master 😞.

kirelagin commented:
So the problem is that Meson is making the `setup.py` output the compiled Python module into the `build/build` directory, but the upstream test is looking for it in the source directory.

kirelagin committed:
pylibfdt: Use the correct Python for tests

By default, Meson will find the version of Python that it itself is
using, i.e. the default one in Nixpkgs.
However, when building libfdt for other Python package sets, we need to
make sure we match the version of the Python used for tests – otherwise
the interpreter will not find the compiled module.

Tweak meson.build to always use the Python interpreter that was passed
to the expression, rather than the default one.

kirelagin commented:
Ok, so, the root of the problem is that `python312Packages.libfdt` builds a `pylibfdt` for Python 3.12, but Meson uses the Python that it was built with to execute the Python tests, which, these days, is Python 3.13.

kirelagin commented:
Fixed in #423299.

kirelagin commented:
> Is it possible for you to rebase this change on top of the fix?

@otavio  Done
